src/components/header/AccountMenuItem.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { SFC } from 'react';
+import styled from '../../styled-components';
+
+import { Link } from '../layout/StyledCopy';
+
+import User from '../icons/User';
+import Book from '../icons/Book';
+import GitBranch from '../icons/GitBranch';
+import Heart from '../icons/Heart';
+import Logout from '../icons/Logout';
+
+interface Props {
+    to?: string;
+    icon: string;
+    onClick?: () => void;
+}
+
+const ICONS = {
```

Comment:
Why not an enum?

Reply:
Could also be an enum, no specific reason